climada/engine/unsequa/calc_base.py

```diff
@@ -231,12 +242,20 @@ def make_sample(self, N, sampling_method='saltelli',
             'names' : param_labels,
             'bounds' : [[0, 1]]*len(param_labels)
             }
-
+        #not sure if we want to still force users to enter a value for N in the
+        #case of the 'ff' sampler. In the meantime, trigger a warning.
```

Comment:
I think I would leave it as is at the moment and just have the warning.
```suggestion
        #for the ff sampler, no value of N is needed. For API consistency the user
        #must input a value that is ignored and a warning is given. 
```